### PR TITLE
Feat: Long-term absence

### PR DESCRIPTION
**Pull request by joannafawl**
#### Work done
- Add `LongTermAbsence` column to worker table in database
- Add `longTermAbsence` to worker endpoint
- Create `long-term-absence` component
- Create `longTermAbsence` endpoint to get enum values from the database

#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ var approvals = require('./server/routes/approvals');
 var satisfactionSurvey = require('./server/routes/satisfactionSurvey');
 var registrationSurvey = require('./server/routes/registrationSurvey');
 var cqcStatusCheck = require('./server/routes/cqcStatusCheck');
+var longTermAbsence = require('./server/routes/longTermAbsence')
```
Comment (aaron-russell) [inline server.js:69]: I really don't like the use of var, but I know this file has that.

File: src/app/features/workers/long-term-absence/long-term-absence.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+  it('should reset radio buttons when pressing back at work checkbox', async () => {
+    const { component, fixture, getByText } = await setup();
+
+    component.worker.longTermAbsence = 'Illness';
+    fixture.detectChanges();
+
+    const backAtWorkCheckbox = getByText('Set as back at work');
+    fireEvent.click(backAtWorkCheckbox);
+
+    expect(component.form.value.longTermAbsence).toBe(null);
```
Comment (aaron-russell) [inline src/app/features/workers/long-term-absence/long-term-absence.component.spec.ts:121]: I would also check the value earlier on to make sure it turns null. If that makes sense?

File: src/app/features/workers/long-term-absence/long-term-absence.component.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    this.workerService.setReturnTo(null);
+  }
+
+  private getReasonsForLongTermAbsence(): void {
+    this.subscriptions.add(
+      this.workerService.getLongTermAbsenceReasons().subscribe((reasons) => {
+        this.longTermAbsenceReasons = reasons;
+      }),
+    );
+  }
```
Comment (aaron-russell) [inline src/app/features/workers/long-term-absence/long-term-absence.component.ts:63]: Could this be in a resolver?

**Review by aaron-russell — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (joannafawl) [reply server.js:69]: Do you think it's better to change this one to `const` or keep it consistent? Or change them all to `const`?

Comment (aaron-russell) [reply server.js:69]: I debate this every time, and I end up doing exactly as you've done, keeping it consitent

**Review by aaron-russell — APPROVED** [review]

**Pull request merged**